.github/workflows/releaseTest.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I understand the need of creating a separate workflow but its seems wasteful for me only to update the RELEASE_TAG? Sure it could be done in the CD (maybe in the future) but for now, would it be better to pass as a input flag such as `isRelease` for testing?

Reply:
I think this is a better practice, because you don't know if a commit will become a release, so it will be challenging to assign a flag to a commit.  Meanwhile, with this method and time a release is made the doesn't have to worry about updating anything. I also don't understand why we should do it in the future since it already is completed and working

Reply:
Step back a little bit: it seems I have not clearly explained my point here. In the future, we will have a separate CI (in charge of integration test and release candidate) and CD ( in charge of validation test and release final version). Therefore, since this workflow is only for updating release tag and for *testing*, would it be better to make it as a temporary script in the `integrationTest` workflow? Its not a one-way door so its up to the others

Reply:
I think this script can stay until the separate CI is develop. I am not sure I understand how you can move it to integrationTest, since how would it be triggered on a release?

Reply:
When will that CI/CD workflow be worked on? It's not hard to switch from one way or the other. There's no reason to block this PR on the eventuality of something that we say we're going to do in the future